projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.ts

```diff
@@ -77,9 +77,98 @@ export class IgxSplitBarComponent {
         return this.siblings[0].hidden && !this.siblings[1].hidden;
     }
 
+    get collapseNextIcon() {
```

Comment:
These getters - collapseNextIcon/collapsePrevIcon are no longer used anywhere and should be removed.